### DISCUSSION TITLE
Renaming `master` to `main`?

### DISCUSSION THREAD

**Discussion by jeska** [microsoft/pyright#1820] [Ideas]
Hi, there!

I am not sure if this qualifies as an issue so I'll address it here. Per the [github/naming](https://github.com/github/renaming) repo & new guidelines, new repositories are being created with the branch `main` instead of `master`. Will `pyright` be renaming its `master` branch in the near future or is it currently not on the road map? I'm sure there might be some up/downstream consequences/complications of changing this 😅 but mostly curious if there's anything in the works.

Thanks!

**Comment by erictraut** [comment]
Thanks for pointing this out. Yes, we should do this. It will require some coordination because of the relationship between pyright and pylance, but it's doable. @jakebailey, let's talk about how and when to make this happen.

**Reply by jeska** [reply]
Amazing, thanks! I'll keep an eye out here. Appreciate the quick response!

**Reply by erictraut** [reply]
@jeska, the rename is complete. Thanks again for the suggestion.

**Reply by jeska** [reply]
Amazing, thank you so much!
